### ISSUE TITLE
AIR SDK since 50.2.3.7 missing to build or package `lib` folder with ANEs

### ISSUE THREAD

**Issue by sjabberwocky** [airsdk/Adobe-Runtime-Support#2966]
I tried to build latest update for our game in 2023 with latest AIR SDK, but it looks like that something was broken after 50.2.3.6 and it impossible package project.

AIR SDK since 50.2.3.7 missing to build or package `lib` folder with ANEs.

It reproduces on 50.2.3.7, 50.2.3.8 and 50.2.4.1.

I use latest IDEA, Android Studio and Windows 11.

@ajwfrost It looks critical from my point of view.

**Comment by ajwfrost** [comment]
Yes -> we have a workaround for it, and the fix will be coming out shortly..
https://github.com/airsdk/Adobe-Runtime-Support/issues/2950#issuecomment-1841552568
thanks
